Main.R

```diff
@@ -45,20 +45,22 @@ execute <- function(jobContext) {
   do.call(CohortDiagnostics::executeDiagnostics, args)
   
   unlink(file.path(exportFolder, sprintf("Results_%s.zip", jobContext$moduleExecutionSettings$databaseId)))
-  unlink(file.path(exportFolder, "database.csv"))
-  unlink(file.path(exportFolder, "cohort.csv"))
   
   moduleInfo <- ParallelLogger::loadSettingsFromJson("MetaData.json")
   resultsDataModel <- readr::read_csv(file = system.file("settings", "resultsDataModelSpecification.csv", package = "CohortDiagnostics"),
                                       show_col_types = FALSE)
+  # AS: Revising some of the column names to match the 
+  # names required by OhdsiSharing. This can be removed once we have
+  # modified this info in CohortDiagnostics
+  names(resultsDataModel)[names(resultsDataModel) == 'fieldName'] <- 'columnName'
```

Comment:
These lines can probably be removed as soon as [this patch](https://github.com/OHDSI/CohortDiagnostics/pull/858) is finished.